web/packages/teleterm/src/ui/DocumentCluster/DocumentCluster.tsx

```diff
@@ -82,8 +82,8 @@ export function Cluster() {
   }
 
   return (
-    <Layout mx="auto" px={5} pt={2} height="100%">
-      <ClusterResources />
+    <Layout mx="auto" px={5} pt={4} height="100%">
+      <UnifiedResources clusterUri={clusterCtx.clusterUri} />
```

Comment:
Yeah, Web UI has some logic that checks whether the server supports listing unified resources.

But since this feature was planned to release in v15, I won't change that and complicate our lives by backporting it to v14 (and having to support v13 clusters).